tsweb/tsweb.go

```diff
@@ -169,7 +169,8 @@ type ReturnHandler interface {
 type HandlerOptions struct {
 	QuietLoggingIfSuccessful bool // if set, do not log successfully handled HTTP requests (200 and 304 status codes)
 	Logf                     logger.Logf
-	Now                      func() time.Time // if nil, defaults to time.Now
+	Now                      func() time.Time           // if nil, defaults to time.Now
+	GenerateRequestID        func(*http.Request) string // if nil, no request IDs are generated
```

Comment:
Maybe if we use a `type RequestID string` type we can put the docs about recommended format in one place

Reply:
Done